Kernel/PerformanceEventBuffer.cpp

```diff
@@ -199,8 +199,15 @@ ErrorOr<void> PerformanceEventBuffer::to_json_impl(Serializer& object) const
 {
     {
         auto strings = object.add_array("strings");
-        for (auto const& it : m_strings) {
-            strings.add(it->view());
+        Vector<KString*> strings_sorted_by_index;
+        strings_sorted_by_index.resize(m_strings.size());
```

Comment:
Please use fail-able vector APIs in new code.

Reply:
Good idea, applied the changes as requested.